packages/accounts/src/kernel-zerodev/README.md

```diff
@@ -35,20 +37,21 @@ const validator: KernelBaseValidator = new KernelBaseValidator({
   owner,
 });
 
+const entryPointAddress = getDefaultEntryPointContract(chain);
+
 // 2. initialize the provider and connect it to the account
 const provider = new KernelAccountProvider(
   // the demo key below is public and rate-limited, it's better to create a new one
   // you can get started with a free account @ https://www.alchemy.com/
   "https://polygon-mumbai.g.alchemy.com/v2/demo", // rpcUrl
```

Comment:
btw, `demo` is going to get heavily rate limited. we should sub this for an actual api key behind an env variable. out of scope here, but if you can make a ticket 🙏

Reply:
this is their README?

Reply:
let's maybe just do:
```suggestion
  "https://polygon-mumbai.g.alchemy.com/v2/<YOUR-ALCHEMY-API-KEY>", // rpcUrl
```